### ISSUE TITLE
[Bug]: WHCM doesn't render icons in text color

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#2229]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

everything with an icon

### Expected behavior

Icons should be rendered in the same colour as their surrounding text

### Actual behavior

icons are rendered as the icon colour

### What browsers are you seeing the problem in?

Chrome, Microsoft Edge

### How can we reproduce this issue?

1. Enable High Contrast mode (or emulation in chrome/edge)
2. Go to https://opensource.adobe.com/spectrum-web-components/components/accordion/
3. See icons are not in the text colour
4. Scroll to https://opensource.adobe.com/spectrum-css/accordion.html
5. See icons are correct in spectrum-css

**Comment by jnurthen** [comment]
I think this is because the svg is in the shadow root whereas spectrum-css applies the class directly to the icon.

**Comment by Westbrook** [comment]
Making some progress in https://spectrum-css--spectrum-web-components.netlify.app/storybook/?path=/story/textfield--default It does seem that we were missing a little style passing into the SVG:

https://github.com/adobe/spectrum-web-components/blob/spectrum-css/packages/icon/src/icon.css#L45-L51

Working through some of @jnurthen's other updates to Spectrum CSS right now, and should have this ready for review later in the week.

**Comment by Westbrook** [comment]
Quick reference:

<img width="799" alt="image" src="https://user-images.githubusercontent.com/1156657/168695482-55684cc5-571f-4768-aa1b-7e3e973300b8.png">

<img width="206" alt="image" src="https://user-images.githubusercontent.com/1156657/168695530-4694003a-768f-4086-af91-a9acdaa02f51.png">
